client/layout/guided-tours/tours/checklist-about-page-tour.js

```diff
@@ -59,15 +59,14 @@ export const ChecklistAboutPageTour = makeTour(
 			arrow="top-left"
 			placement="below"
 		>
-			<Continue target="editor-featured-image-current-image" step="choose-image" click>
-				<p>
-					{ translate(
-						'Featured images are a great way to add more personality to your pages. ' +
-							"Let's add something a little more relevant to your About page text."
-					) }
-				</p>
-				<p>{ translate( 'Press anywhere on this image so we can change it.' ) }</p>
-			</Continue>
+			<p>
+				{ translate(
+					'Featured images are a great way to add more personality to your pages. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Featured images are a great way to add more personality to your pages. Let's add something a little more relevant to your About page text.' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/tour-console-errors)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).